feat: revamping the Navbar #1195

pranay911 opened the pull request:
Changed the color of the navigation item and Added a navigation button according to design.

### 📸 Screenshots 
<!--- (Remove this section if no SS) --> 

![image](https://github.com/milancommunity/Milan/assets/65008142/873e0e21-b8e7-4def-938e-93f204ac91f2)

pranay911 commented:
Hi @tamalCodes I've considered all the approaches you mentioned. In the provided image, there's only one item on the right side. However, on your website, two buttons – "Contribute to Milan" and "Sign Up" – occupy some space. Due to this, the navigation items are not centered as expected.

tamalCodes commented on src/components/Button/GlobalButton/Button.module.css:
```diff
@@ -1,12 +1,12 @@
   text-align: center;
   cursor: pointer;
-  border-radius: 8px;
+  border-radius: 40px;
```
The radius might be different at different places so make another variant that also defines the border radius of the button for example you can make a class called `rounded-button` which has a different border radius and the default button can have the radius that we previously had.

We can use this class in places where we need it.

pranay911 replied:
U mean hide contribute to milan button?

pranay911 replied:
So basically I create a another button with different border-radius...

tamalCodes commented on src/components/Navbar/Navbar.jsx:
```diff
@@ -127,14 +128,20 @@ const Navbar = () => {
                   <>
                     <Button
-                      variant="outline"
+                      variant="Primary"
                       to="/auth/login"
                       fontweight="bold"
                     >
-                      Sign in
+                      <span>
+                        Contribute to Milan
+                        <IoChevronForwardSharp style={{ marginLeft: "5px" }} />
+                      </span>
                     </Button>
```
As discussed, for now let us hide this

tamalCodes replied:
@pranay911 , you said that you would hide it - but it is still visible.
![image](https://github.com/milancommunity/Milan/assets/72851613/16825a6b-bcdf-4f7c-885c-898d9f2e7925)

Also when hovering over the button everything disappears, add a border for the hovering instance. Also please make the navbar texts be of a lesser font-weight, 400 would be great.
![image](https://github.com/milancommunity/Milan/assets/72851613/44e9e5fa-ea4e-4fa2-b264-d384362fbfe4)

pranay911 commented:
Hey @tamalCodes, good evening! Following our discussion, I've made the following changes:
1. Added a new class for rounded buttons.
2. Corrected the border-radius.

pranay911 commented:
Hey @tamalCodes, any updates on PR?

pranay911 commented:
**Now its looks Pretty**

![image](https://github.com/milancommunity/Milan/assets/65008142/8e864407-fb48-4e56-817c-3ff3ae76234a)

tamalCodes reviewed (approved):
# Amazing work 🎉
**The changes look good to me and will be merged soon.**

Do follow [Tamal](https://github.com/tamalCodes) for more Opensource fun projects and don't forget to drop a star so that you get updated about our latest [releases](https://github.com/MilanCommunity/Milan/releases) (we will tag you and mention your work) and also a [shoutout](https://www.linkedin.com/feed/update/urn:li:activity:7073574804519354368/) on social media (LinkedIn and Twitter) !

**PLEASE DROP US A STAR, MEANS ALOT.**

Happy Opensource 🚀.
